### PR TITLE
Update references to old repository name

### LINKED ISSUE #78: Rename project to DistrictBuilder
We should consider referring to this project as DistrictBuilder and changing the name of the older version of this application to DistrictBuilder Classic. GitHub repositories will need to be renamed, references in documentation will need to be updated, and AWS resources will need to be cleaned up.

### PR DESCRIPTION
**Pull request by ColeKettler**
## Overview

Updates all references to DistrictBuilder 2 to reference DistrictBuilder instead. The old repository has been renamed to DistrictBuilder Classic.

### Notes

References / links in documentation, configs, and manifest files have been updated to reflect the new repository name.

No longer maintained notices have been applied to Classic repos in PublicMapping/districtbuilder-classic#560 and PublicMapping/districtbuilder-classic-config#2.

## Testing Instructions

- Verify that the [Jenkins build](http://urbanappsci.internal.azavea.com/job/PublicMapping/job/districtbuilder/job/PR-135/1/) passes.
- Confirm that all references to DistrictBuilder 2 have been replaced with DistrictBuilder.

Closes #78

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ DistrictBuilder 2 is web-based, open source software for collaborative redistric
   - [Stack](#stack)
   - [Ports](#ports)
 - [Scripts](#scripts)
+- [Command Line Interface](#command-line-interface)
```
Comment (ColeKettler) [inline README.md:15]: This is not directly related to this PR, but I noticed that this entry was missing from the ToC while adding an entry for the DistrictBuilder Classic notice. Figured I'd fix it since I was already in there. 🛠️

File: README.md [hunk 2]
```diff
@@ -93,7 +95,11 @@ In order to allow for code-sharing across the frontend and backend in conjunctio
 | `update`    | Build container images, update dependencies, and run database migrations. |
 | `yarn`      | Execute Yarn CLI commands.                                                |
 
-## Command-line interface
+## Command Line Interface
```
Comment (ColeKettler) [inline README.md:98]: Not a compound word or a modifier so I removed the hyphen.

File: README.md [hunk 3]
```diff
@@ -93,7 +95,11 @@ In order to allow for code-sharing across the frontend and backend in conjunctio
...
 
-A command-line interface is available for performing data processing operations.
+A command line interface is available for performing data processing operations.
 See `src/manage/README.md` for more info.
+
+## DistrictBuilder Classic
+
+This project is a continuation of the original version of DistrictBuilder, now called [DistrictBuilder Classic](https://github.com/PublicMapping/districtbuilder-classic). DistrictBuilder Classic is no longer being maintained. This repository is where active development of DistrictBuilder will continue to occur.
```
Comment (ColeKettler) [inline README.md:105]: There's a no longer maintained notice in the (newly renamed) [DistrictBuilder Classic repo](https://github.com/PublicMapping/districtbuilder-classic) that directs folks to this repo instead.

Comment (rbreslow) [reply README.md:105]: I think it'd be worth having Tyler give a 👍 to the language used in both the README in the old repo and this repo.

For this repo, this doesn't feel right at the bottom of the README. I think it makes more sense to expand upon the notice at the top of the document, but that's why I think we should get Tyler's thoughts.

Comment (ColeKettler) [reply README.md:105]: Agreed! @tgilcrest, what are your thoughts on the wording and placement of this notice and the [notice in the Classic repo](https://github.com/PublicMapping/districtbuilder-classic#no-longer-maintained)?

**Comment by ColeKettler** [comment]
Outside of the changes in this PR, I've updated:

- our Codetree repository sync
- the `districtbuilder` (formerly `district-builder-2`) Jenkins job
- the S3 buckets containing Terraform config (done in #65)
- the Terraform project name variable (done in #65)
- the project folders in the sysadmin section of the fileshare (reminded by your comment and https://github.com/PublicMapping/districtbuilder/issues/78#issuecomment-610016434)

Can you think of anything else?

File: README.md (hunk 3 above)
Comment (tgilcrest) [reply README.md:105]: I agree with Rocky, in that I think it makes more sense at the top of the document. The note is short enough that I don't think it needs it's own section at the bottom.

For this notice, I would combine the first and second sentences:
> This project is a continuation of the original version of DistrictBuilder, now called [DistrictBuilder Classic](https://github.com/PublicMapping/districtbuilder-classic), which is no longer being maintained. This repository is where active development of DistrictBuilder will continue to occur.

So, I think the order at the top should be:
- Notice about DB classic
- Notice about how this is early stage
- Add a section heading for ## Overview
- `DistrictBuilder is web-based, open source software for collaborative redistricting.`

I think the notice at the top of the Classic repo looks good. The only (very minor) feedback I have there is that `DistrictBuilder is software created by the Public Mapping Project.` is in the "No Longer Maintained" section header.

**Comment by ColeKettler** [comment]
@rbreslow Last round of suggestions applied, ready for another round!
